docs/structured-streaming-kafka-integration.md

```diff
@@ -401,15 +402,16 @@ The following configurations are optional:
 <tr>
   <td>endingOffsetsByTimestamp</td>
   <td>json string
-  """ {"topicA":{"0": 1000, "1": 1000}, "topicB": {"0": 2000, "1": 2000}} """
+  """ {"topicA":{"0": 1000, "1": 1000}, "topicB": {"0": 2000, "1": 2000}} """ or """ {"topicA":{"-1": 1000}, "topicB": {"-1": 2000} """
   </td>
-  <td>latest</td>
+  <td>the value of <code>endingOffsets</code> will apply</td>
```

Comment:
This is actually same meaning as NOTE 2 but in case of misunderstanding I just mentioned this again here. Please let me know if it is just OK to be latest.